### ISSUE TITLE
FB channel trigger compromised by deprecation on FB Graph Api v8

### ISSUE THREAD

**Issue by odanielduarte** [rapidpro/rapidpro#1333]
Hi,

Upon trying to set up the trigger "start a flow when a conversation is started by a contact", the creation will not be completed and the following error will be raised, showing that the endpoint thread_settings is [deprecated](https://developers.facebook.com/docs/graph-api/changelog/version8.0):

![image](https://user-images.githubusercontent.com/64784188/92279201-30a44600-eecd-11ea-82f0-b8f49ab20e22.png)
 
The issue will occur to any newly created Facebook App, since the 8.0 version or higher is now mandatory in the creation.

Regards,
Daniel Duarte

**Comment by nicpottier** [comment]
@norkans7 Have we seen this on our side?

**Comment by norkans7** [comment]
Yeah, I think I noticed that when adding the new Facebook channel type, and no way I have seen to support the "new conversation" triggers

**Comment by matmsa27** [comment]
Hi @norkans7 and @nicpottier , do you have any news on this?

**Comment by nicpottier** [comment]
We haven't looked at this yet, have you? Any suggestions since you seem to be blocked by this?

**Comment by rasoro** [comment]
Hi there. 

It is currently possible to set up a trigger "start a flow when the conversation is started by a contact" and make it work if you set up the messenger_profile with get_started according to this facebook [documentation](https://developers.facebook.com/docs/messenger-platform/reference/messenger-profile-api/). 

I tested it here and it worked.

![Captura de tela de 2021-08-06 08-33-57](https://user-images.githubusercontent.com/75167930/128504758-236d4b74-020b-4168-91b0-d3b1f9b4b2d2.png)

![Captura de tela de 2021-08-06 08-34-31](https://user-images.githubusercontent.com/75167930/128504775-e4364232-5050-40cb-a0e8-5b043dcaa30b.png)
